File: FIAP.Library.Domain/Entities/RentBook.cs
using FIAP.Library.Domain.Enums;

namespace FIAP.Library.Domain.Entities
{
    public class RentBook
    {
        public int Id { get; set; }
        public int bookId { get; set; }
        public int customerId { get; set; }
        public DateTime datePossibleLocation { get; set; }
        public ERentalStatus status { get; set; }

        public virtual Customer customers { get; set; }
        public virtual Book books { get; set; }
    }
}

File: FIAP.Library.UI/src/app/rentbook.ts
export class RentBook {

    constructor(
      public id: number,
      public rentalDate: string,
      public deletionDate: string,
      public duoDate: string,
      public customerId: number,
      public bookId: number,
      public status?: string
    ) {  }
  
  }
  